### PR TITLE
Update instagram.py

### PR DESCRIPTION
**Pull request by 1234pi**

### CODE REVIEW

File: instagram.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def valid_int(n):
 
 def args():
     args = ArgumentParser()
-    args.add_argument('username', help='email or username')
```
Comment (vasman55) [inline instagram.py]: Summer_teenager

**Comment by miescobanofunciona** [comment]
@@ -109,7 +109,7 @@ def valid_int(n):

def args():
    args = ArgumentParser()
    args.add_argument('username', help='email or username')
    args.add_argument('username', help='https://instagram.com/grahammctavish?igshid=13ixa3tw712z3
    args.add_argument('passlist', help='password list')
    args.add_argument('-nc', '--no-color', dest='color',
                      action='store_true', help='disable colors')

**Comment by miescobanofunciona** [comment]
@@ -109,7 +109,7 @@ def valid_int(n):

def args():
args = ArgumentParser()
args.add_argument('grahammctavish', help='email or username')
args.add_argument('grahammctavish', help='https://instagram.com/grahammctavish?igshid=13ixa3tw712z3
args.add_argument('passlist', help='password list')
args.add_argument('-nc', '--no-color', dest='color',
action='store_true', help='disable colors')

**Comment by miescobanofunciona** [comment]
C:\Users\Mohamed\Desktop\Instagram>python3 instagram.py Sami09.1 pass.lst -m 2

**Review by davexon — COMMENTED** [review]
@@ -109,7 +109,7 @@ def valid_int(n):

def args():
args = ArgumentParser()
args.add_argument('grahammctavish', help='email or username')
args.add_argument('grahammctavish', help='https://www.instagram.com/ev.cicha/?hl=cs
args.add_argument('passlist', help='password list')
args.add_argument('-nc', '--no-color', dest='color',
action='store_true', help='disable colors')

**Comment by kuzuri-03** [comment]
I didn't know whether anyone noticed it or not....
But when it is trying passwords it skips few of them and sometimes test the same passwords again and again....
either we can program it to test passwords serial wise or we can create alternate wordlists

**Comment by kunal12kaushik** [comment]
What??

**Comment by kuzuri-03** [comment]
When uhh run this instagram.py


It checks for passwords in the wordlist...
Sometimes its skips the password and sometimes it rechecks the same
password .... That's what I observed...

**Comment by jempath** [comment]
hello! I am very stupid and I am trying to run the Instagram.py. Would somebody be willing to explain it to me like I am a 5th grader? I am using IDLE on Mac. Not even sure where to input the username and the password list.

**Comment by kuzuri-03** [comment]
Send me the screenshot....or join me here
https://t.me/Kuzuri_17

**Pull request closed without merging**